pkg/utils/config_test.go

```diff
@@ -302,8 +261,8 @@ func TestFetchListingAddresses(t *testing.T) {
 		t.Errorf("Should have fetched listing addresses: len: %v, err: %v",
 			len(config.ContractAddresses["newsroom"]), err)
 	}
-	// 1 newsroom from config, 14 from graphql
-	if len(config.ContractAddressObjs["newsroom"]) != 15 {
+	// 1 newsroom from config, 7 from graphql
+	if len(config.ContractAddressObjs["newsroom"]) != 8 {
```

Comment:
Need a test for the fetch multisig method?

Reply:
sure, i'll add one